secrets/secrets-vault/README.md

```diff
@@ -0,0 +1,119 @@
+
+## How to use it
+
+### Install with Helm
+
+If you are looking to run Secrets Vault Server as part of an existing Kubernetes-based deployment, you can easily deploy it using the bundled [Helm chart][helm-chart] using the following command:
+
+```shell
+$ helm install <NAME> oci://ghcr.io/wasmcloud/charts/secrets-vault
+```
+
+For detailed information on the available configuration options, please see the [Helm chart README][helm-chart].
+
+### Configuring Vault
+
+In order for Secrets Vault Server to work as intended, it will need the following to be pre-configured on the Vault Server it's talking to:
+
+1. Vault Server will need to an instance of the [JWT auth method enabled][vault-jwt-auth-enabled] specifically for the Secrets Vault Server.
+2. The JWT auth method will need to be configured to with the [`jwks_url`][vault-jwks-url] configured to point at the JWKS endpoint exposed by the Secrets Vault Service (configured via `--jwks-address` flag `SV_JWKS_ADDRESS` environment variable). Optionally you can also include a default_role for the backend
+
+An example of configuring the above steps might look like this:
+
+```shell
+# Enable jwt auth method at provided path
+$ vault auth enable -path=jwt jwt
+
+# Configure jwt auth to point it's jwks_url at the JWKS endpoint provided by the Secrets Vault Server
+# Please note that the Secrets Vault Server needs to be running in order for Vault to verify the endpoint.
+$ vault write auth/jwt/config jwks_url="http://localhost:3000/.well-known/keys"
+
+# Create a named role with configuration from demo-role.json, see below for example.
+$ vault write auth/jwt/role/demo-role @demo-role.json 
+```
+
+Example `demo-json.role`:
+
+```json
+{
+  "role_type": "jwt",
+  "policies": ["demo-role-policy"],
+  "bound_audiences": "Vault",
+  "bound_claims": {
+    "application": ["rust-hello-world", "rust-http-kv", "tinygo-hello-world"]
+  },
+  "user_claim": "sub"
+}
+```
+
+Once you have enabled the jwt auth method and created a named role, you will also need to create the `demo-role-policy` policy:
+
+```shell
+# Create the policy named demo-role-policy with the contents from stdin:
+$ vault policy write demo-role-policy - << EOF
+# Dev servers have version 2 of KV secrets engine mounted by default, so will
+# need this path to grant permissions:
+path "secret/data/*" {
+  capabilities = ["create", "update", "read"]
+}
+EOF
+```
+
+With the role created and configured, you can now reference any secrets you write on the default `secret` path.
+
+To configure a wadm application to use policies, you will need to add `policies` section in your wadm manifest:
+
+```yaml
+spec:
+  policies:
+    - name: vault-example
+      type: secrets.wasmcloud.dev/v1/vault
```

Comment:
```suggestion
      type: policy.secret.wasmcloud.dev/v1alpha1
```

Reply:
Addressed it before I saw your feedback, thanks for calling it out though!